plugins/drupal-plugin/src/components/brand-logo.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
```

Comment:
Not at all blocking, but I think using `git mv` for files like this would've reduced the number of lines changed in the PR

Reply:
There's actually a `brand-logo.js` still there in the directory as well. I sort of took the quick route of duplicating these CMS specific component files rather than dealing with transpiling them in the `publish.js` script. I could make this smarter and save a few files, but didn't seem worth the trouble since each output dist repos will look the same either way.